go/client/cmd_config.go

```diff
@@ -103,11 +103,25 @@ func (v *CmdConfigSet) ParseArgv(ctx *cli.Context) error {
 			return fmt.Errorf("Missing string value argument")
 		}
 		s := args[1]
+		if v.looksLikeBool(s) {
```

Comment:
but now it's impossible to do `keybase config set bar t` if you wanted to.

I think `keybase config set -s bar t` should still work...